src/dotnet/UI.Blazor/Components/EscapeHandler/escape-handler.ts

```diff
@@ -1,26 +1,28 @@
-export class EscapeHandler {
-    private _blazorRef: DotNet.DotNetObject;
-    private _elementRef: HTMLDivElement;
+import { fromEvent, Subject, takeUntil} from 'rxjs';
```

Comment:
Just read https://rxjs.dev/guide/subscription - overall, no need to work on this, rxjs subscription API has all I was thinking about here (i.e. unsubscribing w/o specifying any details via `unsubscribe` & joining subscriptions via `add`).